tests/Traits/DataHelpers/ProductHelper.php

```diff
@@ -31,7 +31,8 @@ public function createVisibleTestProduct(array $overrides = [], bool $withVarian
     public function createTestProduct(
         array $overrideData = [],
         bool $withVariant = false,
-        bool $overrideRecursively = false
+        bool $overrideRecursively = false,
+        bool $withManufacturer = true
```

Comment:
Wouldn't it be easier to use the `$overrideData`? I want to avoid we add more parameters to this, as a method call already looks completely wild when using all parameters:

```php
$this->createTestProduct([], false, false, false);
```

Without checking the code or method arguments you would have no idea whats going on. If you want to easily create products without a manufacturer, you could add a method with the name `createTestProductWithoutManufacturer` or similar.

Reply:
Setting the manufacturer `null` inside the data array leads to an error while creating:
![image](https://user-images.githubusercontent.com/51320851/157006502-2b1414ee-735c-47fe-8ca6-a98c9c46c4ce.png)

Reply:
We need to find a solution for this in the future. Otherwise we'ill end up with a method that is unreadable and hardly usable. A builder pattern would be the best approach here. For now we can leave it like that.